packages/common/index.ts

```diff
@@ -805,7 +808,9 @@ export const ERROR_MESSAGES = {
       cannotCheckIntoMultipleQueues:
         "Cannot check into multiple queues at the same time",
     },
+    courseAlreadyRegistered: "The course was already registered",
```

Comment:
We made the front end to directly display these error messages in the case of an error, so let's make them a bit more user friendly and say something like, "One or more of the courses is already registered". Same goes for `sectionGroupNotFound`, let's make it account for the case where the prof is registering multiple courses as well.